### PR TITLE
Profile shashi

### PR DESCRIPTION
**Pull request by shashiBhushan13**
Add my resume to your folder

### CODE REVIEW

File: batches/2020/career-camp-06-january-2020/index.html [hunk 1]
```diff
@@ -93,6 +93,22 @@
...
+    <div class="col">
+      <div class="card card-width">
+        <a href="./students/shashi-resume/index.html" target='_blank' class="no-text-decoration">
+            <div class="img-height-div">
+            <img src="./students/shashi-resume/ser.jpg" class="card-img-top" alt="profile-pic">
+            </div>
+            <div class="card-body pr-0 pl-0 pt-0 pb-0 card-body-student overflow-auto">
+            <p class="card-text text-center">
+                <span class="black">Shashi</span>
+                <br><small class="small-font grey ">.</small>
```
Comment (dakshkhetan) [inline batches/2020/career-camp-06-january-2020/index.html:105]: Please add a mini-bio (refer other cards) in place of dot (.)

**Review by dakshkhetan — CHANGES_REQUESTED** [review]

**Comment by shashiBhushan13** [comment]
Added mini-bio,merge it please

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
I guess you haven't pushed the commits. Please do that. As there is still a . (dot) appearing in your mini-bio in this PR.

**Comment by dakshkhetan** [comment]
Closing this PR due to no response.
Raise a new one with requested changes done.

**Pull request closed without merging**